### ISSUE TITLE
Container element is empty (empty structure box in a section)

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5418]
Sites identified: Archivo
Page(s) identified: media object page

WCAG compliance level: A

Description:

Example appears to be an empty div in the structure area underneath the section with an empty \<li\> in it.

Some roles are designed to contain other roles. This element has this type of role, but it doesn't contain any required-owned elements.

Roles provide information about content structure and how page elements fit together.

Some roles depend on other roles for context. For example, the role listitem can only really be meaningful in the context of a list or group.

Example record: https://archivomesoamericano.org/media_objects/gf06g267d

See more details at [SiteImprove example](https://my2.siteimprove.com/Inspector/1166400/A11Y/Page?pageId=38027899106&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r68/failed/SC9Hd1pZZm0=;ZGYyOGYxYjI=).

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNUVnQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3eee350e38cf7cd1c45ff8b027015d0fd79d61a3/image.png)

**Comment by elynema** [comment]
This will no longer be an issue once 7.7 is rolled out, but remediation is due for this in early February, so Jon D. would like us to fix this in place if possible with limited effort.

**Comment by joncameron** [comment]
@joncameron investigate on Archivo (could be remedied if it's just one item)

**Comment by joncameron** [comment]
Empty structure removed from https://archivomesoamericano.org/media_objects/gf06g267d. This can probably be closed now.

**Comment by elynema** [comment]
@joncameron SiteImprove hasn't caught up with your change yet, but it also flagged this record: https://archivomesoamericano.org/media_objects/p5547r37h. Can you fix that one as well? I'm not sure I have an edit account on Archivo.

**Comment by joncameron** [comment]
Yep, I removed the empty structure for that item as well.
